crates/nu-command/src/platform/input.rs

```diff
@@ -73,6 +73,11 @@ impl Command for Input {
                     }
                     buffer.push(buf[0]);
 
+                    if buf.contains(&3) {
```

Comment:
I had to look up what `3` was [in this context](https://en.wikipedia.org/wiki/End-of-Text_character). Mind adding a comment explaining the magic number?

Reply:
I could've sworn that I had, but must've undo'd one too many times! Should be added now!